engine/client/ref_common.c

```diff
@@ -600,7 +600,7 @@ void R_CollectRendererNames( void )
 		dll = COM_LoadLibrary( temp, false, true );
 		if( !dll )
 		{
-			Con_Reportf( "R_CollectRendererNames: can't load library %s: %s\n", temp, COM_GetLibraryError() );
+			Con_Reportf( "R_CollectRendererNames: can't load library %s: %s", temp, COM_GetLibraryError() );
```

Comment:
It's not a good idea to remove newline here.

Keep it, if COM_GetLibraryError puts newline, then it should be fixed.

Reply:
Done